tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/InlinePassImpl.scala

```diff
@@ -65,10 +65,17 @@ class InlinePassImpl @Inject()(val options: PassOptions,
     // Remove the operators that are not needed,
     // as some of them may contain higher-order operators that cannot be substituted
     val relevantOperators = NormalizedNames.userOperatorNamesFromOptions(options).toSet
```

Comment:
why don't you extend `userOperatorNamesFromOptions` with the operators for `cinit`, `initPrime`, etc.? These names should keep the relevant operators, so later passes would not remove them.